tests/snapshots/tests.contrib.mariadb.test_mariadb.test_simple_malformed_query_snapshot_post_1_1.json

```
@@ -0,0 +1,32 @@
+[[
+  {
+    "name": "mariadb.query",
+    "service": "mariadb",
+    "resource": "SELEC 1",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "sql",
+    "error": 1,
+    "meta": {
+      "db.name": "test",
+      "db.user": "test",
+      "error.msg": "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'SELEC 1' at line 1",
```

```suggestion
      "error.msg": "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'SELEC 1' at line 1",
```
I think we need to update this file too.